### PR TITLE
add tshark package

### PR DESCRIPTION
**Pull request by jamonation**
#### For new package PRs only

  - [X] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

#### For new version streams

- [X] The upstream project actually supports multiple concurrent versions.
- [X] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

### CODE REVIEW

File: tshark.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package:
+  name: tshark
+  version: 4.4.2
+  epoch: 0
+  description: "TShark is a network protocol analyzer to ump and analyze network traffic"
```
Comment (jamonation) [inline tshark.yaml:5]:
```suggestion
  description: "TShark is a network protocol analyzer to dump and analyze network traffic"
```

**Review by kranurag7 — COMMENTED** [review]
looking at package contents, I think we can do a `split/doc` and `split/dev` as well. 💭

**Comment by jamonation** [comment]
> looking at package contents, I think we can do a `split/doc` and `split/dev` as well. 💭

Ah I tried split/manpages and got null 🤦 

split/dev isn't helpful here - it ends up grabbing the .so symlinks. Since tshark doesn't install any development headers, that subpackage shouldn't be needed.

**Review by kranurag7 — DISMISSED** [review]

File: linkerd2.yaml [hunk 2]
```diff
@@ -176,6 +176,28 @@ subpackages:
           mkdir -p ${{targets.contextdir}}
           ln -sf /usr/bin/tap ${{targets.contextdir}}/tap
 
+  # the linkerd debug image doesn't have linkerd2 binaries, but has a number of tools
```
Comment (jamonation) [inline linkerd2.yaml:179]: Need an epoch bump to get this

**Review by kranurag7 — APPROVED** [review]

**Pull request merged**